core/src/test/java/org/apache/iceberg/TestPartitionStatsUtil.java

```diff
@@ -370,16 +371,167 @@ public void testPartitionStatsWithSchemaEvolution() throws Exception {
             snapshot2.snapshotId()));
   }
 
-  private static PartitionData partitionData(Types.StructType partitionType, String c2, String c3) {
-    PartitionData partitionData = new PartitionData(partitionType);
-    partitionData.set(0, c2);
-    partitionData.set(1, c3);
-    return partitionData;
+  @Test
+  @SuppressWarnings("MethodLength")
+  public void testPartitionStatsWithSchemaEvolution2() throws Exception {
```

Comment:
Could we add a better name for the tests which describes the tested case better?

Reply:
renamed